public/docs/dart/latest/cheatsheet.jade

```diff
@@ -1,6 +1,10 @@
 - var base = current.path[4] ? '.' : './guide';
 .banner.grid-fluid
-  p.text-body.c10 This cheat sheet is provisional and may change. Angular 2 is currently in Release Candidate.
+  .alert.is-important
+    :marked
+       **Known issue**: some cheat sheet entries may be inaccurate since they
```

Comment:
I usually include the `:` in the boldface and capitalize the character afterward (`Some`). Also, I think we should explain/promise less. Maybe:

**Known issue:** Some cheat sheet entries are currently inaccurate, reflecting TypeScript instead of Dart.